src/DOM/Event/DataTransfer.purs

```diff
@@ -0,0 +1,9 @@
+module DOM.Event.DataTransfer where
```

Comment:
Perhaps this should be under `DOM.Event.DragEvent.DataTransfer`? I think it only applies to the drag API anyway.